addons/project/wizard/project_report_wizard_views.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="project_report_wizard_form" model="ir.ui.view">
+        <field name="name">project.report.wizard.form</field>
+        <field name="model">project.report.wizard</field>
+        <field name="arch" type="xml">
+            <form string="Project">
+                <group>
+                    <group>
+                        <field name="project_id"/>
+                    </group>
+                </group>
+                <footer>
+                    <button string="Go to Project Updates"  name="action_project_update_report_view" type="object" class="btn-primary" invisible="context.get('is_burndown', False)"/>
+                    <button string="Go to Burndown Chart" name="action_burndown_report_view" type="object" class="btn-primary" invisible="not context.get('is_burndown', False)"/>
+                    <button string="Discard" class="btn-secondary" special="cancel"/>
+                </footer>
+            </form>
+        </field>
+    </record>
+
+    <record id="project_update_all_report_action" model="ir.actions.act_window">
+        <field name="name">Select a Project</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">project.report.wizard</field>
+        <field name="view_mode">form</field>
+        <field name="view_id" ref="project_report_wizard_form"/>
+        <field name="target">new</field>
+    </record>s
+     <record id="project_burndown_chart_report_action" model="ir.actions.act_window">
+        <field name="name">Select a Project</field>
+        <field name="res_model">project.report.wizard</field>
+        <field name="view_mode">form</field>
+        <field name="view_id" ref="project_report_wizard_form"/>
+        <field name="context">{'is_burndown': True}</field>
```

Comment:
@xavierbol , There is two separate menus for burndown and project update so even for default_report_type_field we have to add the context. so it shouldn't be better to direct access the context instead of a new field.

Reply:
Indeed

Reply:
I think there is no better solution than other here. The both solutions are good. ^^
We will see with @IT-Ideas if he prefers using a new field for that or with the context is sufficient. 🙂

Reply:
Hello @mba-odoo,
I would also prefer not to use a new custom context key, which the could have possibly, one day, a side effect. Using the standard default_FIELDNAME: value look nicer to me.

Thanks 🙂